charts/karmada/values.yaml

```
@@ -134,7 +134,10 @@ certs:
   mode: auto
   auto:
     ## @param certs.auto.expiry expiry of the certificate
+    ## Note: The expiry value should not exceed the rootCA expiry time (rootCADaysExpiry * 24h)
     expiry: 43800h
+    ## @param certs.auto.rootCADaysExpiry expiry of the root CA certificate in days, defaults to 1825 days (5 years)
+    rootCADaysExpiry: 1825
```

```suggestion
    ## @param certs.auto.rootCAExpiryDays expiry of the root CA certificate in days, defaults to 1825 days (5 years)
    rootCAExpiryDays: 1825
```
I prefer the name `rootCAExpiryDays`. What do you think? @ryanwuer